hooks/render_media.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) 2019 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+
+import sgtk
+import tempfile
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class RenderMedia(HookBaseClass):
+    def render(
```

Comment:
This method looks like something that should not be here. There is no default implementation for render, right?

Reply:
Oups !